.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+- Create a minimal reproduction project using the scaffolding tool: `npm init @holochain`
+- Produce the error
+- Upload the project to a repository and link it here
+- Provide steps/commands to reproduce the error
```

Comment:
I wonder if this should move below System Info as it may create a high barrier for some people to report bugs that might still be useful/valid. To answer this, I think we need to know whether we intend to reduce bug reporting by having this barrier in place (and featured so early in the bug report data), or whether we want to know about more bugs even if someone cannot provide the means for us to reproduce the problem.

Reply:
@artbrock Yes, that intention needs to be clarified.

I've seen issues that turned out to be misconfigurations or bugs in zomes. We don't have a standard way of setting up hApp projects AFAIK. I presume that people just copy existing repos to base their own projects on, and it would be great to promote a standard layout with build commands and folder structure correctly wired up.

Preparing for an increasing usage of Holochain, it can make sense to raise the barrier for reporting bugs. You guys need to decide if that's applicable already.